.github/actions/spelling/line_forbidden.patterns

```diff
@@ -1,119 +1,289 @@
+# Should be `prepopulate`
+[Pp]re[- ]populate
+
+# Should be `prerequisite`
+[Pp]re[- ]requisite
+
+# Should be `recently changed` or `recent changes`
 [Rr]ecent changed
 
-# s.b. reentrancy
+# Should be `reentrancy`
 [Rr]e[- ]entrancy
 
-# s.b. reentrant
+# Should be `reentrant`
 [Rr]e[- ]entrant
 
-# s.b. understand
+# Should be `strong suit`
+\b(?:my|his|her|their) strong suite\b
+
+# Should be `understand`
 \bunder stand\b
 
-# s.b. workarounds
+# Should be `URI` or `uri` unless it refers to a person named `Uri`
+#(?<!\.)\bUri\b(?![(])
+
+# Should be `it uses is`
+/\bis uses is\b/
+
+# Should be `uses it as`
+(?:^|\. |and )uses is as (?!an?\b|follows|livestock|[^.]+\s+as\b)
+
+# Should be `was`
+\bhas been(?= removed in v?\d)
+
+# Should be `where`
+\bwere they are\b
+
+# should be `vCenter`
+\bV[Cc]enter\b
+
+# Should be `VM`
+\bVm\b
+
+# Should be `workarounds`
 #\bwork[- ]arounds\b
 
-# s.b. workaround
+# Should be `workaround`
 (?:(?:[Aa]|[Tt]he|ugly)\swork[- ]around\b|\swork[- ]around\s+for)
 
-# s.b. (coarse|fine)-grained
+# Should be `worst`
+(?i)worse-case
+
+# Should be `you are not` or reworded
+\byour not\b
+
+# Should be `(coarse|fine)-grained`
 \b(?:coarse|fine) grained\b
 
-# s.b. neither/nor -- or reword
-#\bnot\b[^.?!"/(]+\bnor\b
+# Homoglyph (Cyrillic) should be `A`/`B`/`C`/`E`/`H`/`I`/`I`/`J`/`K`/`M`/`O`/`P`/`S`/`T`/`Y`
+# It's possible that your content is intentionally mixing Cyrllic and Latin scripts, but if it isn't, you definitely want to correct this.
+(?<=[A-Z]{2})[АВСЕНІӀЈКМОРЅТУ]|[АВСЕНІӀЈКМОРЅТУ](?=[A-Z]+(?:\b|[a-z]+)|[a-z]+(?:[^a-z]|$))
+
+# Homoglyph (Cyrillic) should be `a`/`b`/`e`
+# It's possible that your content is intentionally mixing Cyrllic and Latin scripts, but if it isn't, you definitely want to correct this.
+[аве](?=[A-Za-z]{2,})|(?<=[A-Za-z]{2})[аве]|(?<=[A-Za-z])[аве](?=[A-Za-z])
+
+# Should be `neither/nor` -- or reword
+#(?!<do )\bnot\b([^.?!"/(](?!neither|,.*?,))+\bnor\b
 
-# probably a double negative
-# s.b. neither/nor (plus rewording the beginning)
-\bnot\b[^.?!"/]*\bneither\b[^.?!"/(]*\bnor\b
+# Should be `neither/nor` (plus rewording the beginning)
+# This is probably a double negative...
+\bnot\b[^.?!"/(]*\bneither\b[^.?!"/(]*\bnor\b
 
-# In English, it is generally wrong to have the same word twice in a row without punctuation.
-# Duplicated words are generally mistakes.
-# There are a few exceptions where it is acceptable (e.g. "that that").
-# If the highlighted doubled word pair is in a code snippet, you can write a pattern to mask it.
-# If the highlighted doubled word pair is in prose, have someone read the English before you dismiss this error.
+# In English, duplicated words are generally mistakes
+# There are a few exceptions (e.g. "that that").
+# If the highlighted doubled word pair is in:
+# * code, write a pattern to mask it.
+# * prose, have someone read the English before you dismiss this error.
 \s([A-Z]{3,}|[A-Z][a-z]{2,}|[a-z]{3,})\s\g{-1}\s
```

Comment:
Fwiw, this is the warning for duplicate words... It's currently limited to words of at least 3 characters in length. That could be tweaked if it turns out people make the same mistake I did. I haven't really experimented with decreasing the word length. In general, I've adjusted most rules to only look at 3+ letter words because I found there were too many false positives for 2 letter words. (There are some exceptions in this file...)